src/dec128.rs

```diff
@@ -982,6 +992,14 @@ mod tests {
     fn default() {
         assert_eq!(d128::zero(), d128::default());
         assert_eq!(d128::zero(), Default::default());
+
+        assert!(d128::infinity().is_infinite());
```

Comment:
This function tests default(). Can you add another test called `special()` that tests special values, like infinite, etc?

Reply:
Done